roles/matrix-maubot/templates/maubot_config.yaml.j2

```diff
@@ -0,0 +1,99 @@
+# Other DBMSes supported by SQLAlchemy may or may not work.
+# Format examples:
+#   SQLite:   sqlite:///filename.db
+#   Postgres: postgres://username:password@hostname/dbname
+database: {{ matrix_maubot_database_url }}
+
+plugin_directories:
+    # The directory where uploaded new plugins should be stored.
+    upload: {{ matrix_maubot_container_data_dir }}/plugins
+    # The directories from which plugins should be loaded.
+    # Duplicate plugin IDs will be moved to the trash.
+    load:
+    - {{ matrix_maubot_container_data_dir }}/plugins
+    # The directory where old plugin versions and conflicting plugins should be moved.
+    # Set to "delete" to delete files immediately.
+    trash: {{ matrix_maubot_container_data_dir }}/trash
+    # The directory where plugin databases should be stored.
+    db: {{ matrix_maubot_container_data_dir }}/plugins
+
+server:
+    # The IP and port to listen to.
+    hostname: {{ matrix_maubot_management_hostname }}
+    port: {{ matrix_maubot_management_port }}
+    # The base management API path.
+    base_path: /_matrix/maubot/v1
+    # The base path for the UI.
+    ui_base_path: /_matrix/maubot
+    # Override path from where to load UI resources.
+    # Set to false to using pkg_resources to find the path.
+    override_resource_path: /opt/maubot/frontend
+    # The base appservice API path. Use / for legacy appservice API and /_matrix/app/v1 for v1.
+    appservice_base_path: /_matrix/app/v1
+    # The shared secret to sign API access tokens.
+    # Set to "generate" to generate and save a new token at startup.
+    unshared_secret: generate
+
+{% if maubot_shared_secret %}
+# Shared registration secrets to allow registering new users from the management UI
+registration_secrets:
+    {{ matrix_domain }}:
+        # Client-server API URL
+        url: {{ matrix_maubot_cs_url }}
+        # registration_shared_secret from synapse config
+        secret: {{ matrix_maubot_registration_shared_secret }}
+{% endif %}
+
+# List of administrator users. Plaintext passwords will be bcrypted on startup. Set empty password
+# to prevent normal login. Root is a special user that can't have a password and will always exist.
+admins:
+    root: ""
+{% if matrix_maubot_admins %}
+{% for user, password in matrix_maubot_admins %}
+    {{ user }}: '{{ password }}'
+{% endfor %}
+{% endif %}
+
+# API feature switches.
+api_features:
+    login: true
+    plugin: true
+    plugin_upload: true
+    instance: true
+    instance_database: true
+    client: true
+    client_proxy: true
+    client_auth: true
+    dev_open: true
+    log: true
+
+# Python logging configuration.
+#
+# See section 16.7.2 of the Python documentation for more info:
+# https://docs.python.org/3.6/library/logging.config.html#configuration-dictionary-schema
+logging:
+    version: 1
+    formatters:
+        precise:
+            format: "[%(asctime)s] [%(levelname)s@%(name)s] %(message)s"
+    handlers:
+        file:
```

Comment:
We should probably disable file-logging (like we do for all our other services) and just rely on systemd-journald (console logging).

`/var/log` is not mounted in the container, so we won't be dirtying up the filesystem right now, but it will be making the container size grow (for now good reason). Let's just get rid of this and adjust `handlers: [file, console]` below?